### ISSUE TITLE
Create a .gns3appliance from GNS3

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-gui#1221]
It would be nice to create a .gns3appliance file from an existing template in the preferences. This would make contributions a lot easier.

**Comment by danieldemers** [comment]
Is the point to create a template others could follow? Allowing the community to contribute templates for platforms we want to use?

**Comment by grossmj** [comment]
Yep, that is the point :)

**Comment by b-ehlers** [comment]
I've created an `export_template` tool, that exports a template as a GNS3 appliance. It is integrated into my tools system to extend the GUI https://github.com/GNS3/gns3-gui/issues/708#issuecomment-529929063.

The source is available at <https://git.bernhard-ehlers.de/ehlers/gns3-tools/src/branch/master/export_template>.

**Comment by grossmj** [comment]
Yes please, I think we have an issue open about something similar.

**Comment by grossmj** [comment]
I will add this in v3.1 👍
